### ISSUE TITLE
Dashboard's top bar is missing focus styles

### ISSUE THREAD

**Issue by enriquesanchez** [ParabolInc/parabol#5259]
## Issue - Bug

<img width="1050" alt="Screen Shot 2021-08-11 at 11 07 54 a m" src="https://user-images.githubusercontent.com/3276087/129064453-0eb4c298-7167-4aa1-a9ed-4144c9739090.png">

All of the elements in the app's dashboard are interactive yet not one of them has a focus style to indicate keyboard focus.

A few of them, like the `logo`, the `Search` submit button, and the `user` menu are no focusable at all. They can only be interacted with if the user is using a mouse.

Users who's primary way of navigating is the keyboard have no way of knowing their position when interacting with the dashboard.

Related WCAG success criteria:
- [2.4.7 Focus Visible](https://www.w3.org/WAI/WCAG21/quickref/#focus-visible)
- [2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/quickref/#keyboard)

**How serious is the bug?** P3
- **OS:** All
- **Browser:** All

### Acceptance Criteria 
Users can:
   - Operate with a keyboard all of the interactive items in the app's dashboard. 
   - Know what their focus position is at all times in the app's dashboard

### Suggestions

I strongly suggest we stick to semantic HTML for our interactive components. The `a`, `button` elements already provide the necessary interactive and a11y features that are standard and expected.

### Estimated effort

1 hour to triage. More if root cause is already identified.

**Comment by ackernaut** [comment]
We probably need hover and or focus states designed for this? We may need to add some design AC.

**Comment by enriquesanchez** [comment]
@acressall and I have been looking into this and came up with a focus style that has enough contrast against the top bar's dark purple background, and also stays consistent with any focus styles we already use elsewhere in the app.

We decided to use a border of `Sky 400 #61B1EB` as it's the shade that has enough ratio:

![imagen](https://user-images.githubusercontent.com/3276087/134384161-0421108e-2173-498f-9a75-2cab6db768c6.png)

Note: image above shows focus styles applied to _all_ elements of the top bar for reference. Normally only one item would receive focus at a time. 

Next steps:

- Implement these updates
